flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
Is it on purpose it's limited to x86_64?

Reply:
Yes, it fails to build https://buildbot.flathub.org/#/builders/20/builds/6948